recipes/rsirius/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.9" %}
+{% set name = "r-sirius-ms" %}
```

Comment:
For an R package named `Rsirius`, the convention would be to name the corresponding conda package `r-rsirius`. However, I appreciate that this is a generated API package for interacting with the conda package `sirius-ms`, so I also see the advantage to your users of naming it `r-sirius-ms`. Regardless which name you choose though, the directory name should match the package name